pkg/server/server_test.go

```diff
@@ -37,3 +41,31 @@ func TestServerHealthz(t *testing.T) {
 		t.Errorf("expected status code %d, got %d", http.StatusOK, w.Code)
 	}
 }
+
+func TestServerAccessLog(t *testing.T) {
```

Comment:
This test looks good to me as a functional test. We can follow up with true end to end tests for StoneSoup as a follow up.